---
title: "SRA Results"
output: 
  html_document:
    toc: true
    toc_float: true
---

```{r include=FALSE}
library(plotly)
library(tidyverse)
library(knitr)
library(rmarkdown)
library(patchwork)
library(ggfittext)
library(ggplotify)
library(Matrix)
library(grid)
```

```{r include=FALSE}
snr <- read_delim("./data/final_36_workflows_sra_naive-std_results_auroc_auprc.tsv",
                  delim = "\t", col_names = T)
skr <- read_delim("./data/final_36_workflows_sra_knowledge-std_results_auroc_auprc.tsv",
                  delim = "\t", col_names = T)
```

The results on this page are based on 256 SRA datasets. After preprocessing each dataset using lenient filters in order to keep data for as many genes and samples as possible, we compared methods commonly used in RNA-seq analysis to effectively construct one coexpression network per dataset (i.e. building 256 SRA networks).

Key stages of data processing and network building tested here:
1. Within-sample normalization: counts per million (CPM), transcripts per million (TPM), and reads per kilobase per million (RPKM),
2. Between-sample normalization: quantile (QNT), trimmed mean of M-values (TMM), and upper quartile (UQ), and
3. Network transformation: weighted topological overlap (WTO) and context likelihood of relatedness (CLR).

Individual methods are in regular font (e.g. TPM normalization) and workflows are italicized (e.g. TPM, which TPM combined with no between-sample normalization and no network transformation, or TPM_CLR, which is TPM paired with just CLR).

The evaluation metrics are:

- **Log~2~(auPRC/prior)**; auPRC is the area under the Precision-Recall Curve, prior is the expected auPRC from a random classifier. Focus on this metric to pick the workflow that results in an overall high-quality coexpression network.

- **auROC**; area under the ROC Curve. This commonly-used metric also points to overall quality of the network but is not well-suited for imbalanced problems such as predicting gene functional relationships. We include this metric here for completion.

-  **Log~2~(p20r/prior)**; p20r is precision  at 20% recall. Focus on this metric if you are interested in applying a threshold to the coexpression network and select high-scoring gene pairs for some subsequent analysis.

The evaluations are based on:

- **Naive standard**: Experimentally-verified gene co-annotations to specific Biological Process terms in the Gene Ontology, and

- **Tissue-aware standard**: The naive gold standard subset by genes known to be expressed in a given tissue.

# Data
```{r include=FALSE}
sra_ts_tissues <- unique(skr$tissue)
data_sra <- snr %>% 
  filter(method == "counts") %>% 
  select(project, tissue, sample_num)

data_sra_tissue_projects <- data_sra %>% 
  group_by(tissue) %>% 
  tally() %>%  
  mutate(ts = tissue %in% sra_ts_tissues)

sra_projects_per_tissue_fig <- data_sra_tissue_projects %>% 
  ggplot(aes(x = reorder(tissue, n), y = n, fill = ts)) +
  geom_hline(yintercept = 10, color = "#999999", size = 0.1) +
  geom_hline(yintercept = 20, color = "#999999", size = 0.1) +
  geom_hline(yintercept = 30, color = "#999999", size = 0.1) +
  geom_hline(yintercept = 40, color = "#999999", size = 0.1) +
  geom_col() +
  scale_fill_manual(values = c("light gray", "#2171b5")) +
  scale_y_continuous(limits = c(0,40), breaks = c(0, 10, 20, 30, 40),
                     labels = c("0", "10", "20", "30", "40")) +
  ylab("Number of experiments") +
  geom_text(aes(label = n), hjust = -0.2) +
  coord_flip() +
  theme(panel.background = element_rect(fill = "white"),
        text = element_text(size = 14),
        axis.title.x = element_text(size = 10),
        axis.title.y = element_blank()) +
  guides(fill = FALSE)

sra_data_tissue_order <- data_sra_tissue_projects %>% 
  arrange(n) %>% pull(tissue)
#divide projects into bins according to number of samples
data_sra$bin <- cut(data_sra$sample_num, 
                    breaks = c(4,6,10,16,30,270),
                    labels = c("5-6", "7-10", "11-16", "17-30", "31-270"))
data_sra <- data_sra %>% 
  group_by(tissue, bin) %>% 
  tally()
#add back missing combinations that had zero counts
data_sra_full_na <- tibble(tissue = rep(unique(snr$tissue), 5),
                           bin = c(rep("5-6", 19), rep("7-10", 19),
                                   rep("11-16", 19), rep("17-30", 19),
                                   rep("31-270", 19)), 
                           n = NA)
data_sra_full_na$bin <- factor(data_sra_full_na$bin, levels = c("5-6","7-10","11-16","17-30","31-270"))
data_sra_missing <- anti_join(data_sra_full_na, data_sra, by = c("tissue", "bin"))
#get zero labels for NAs
data_sra <- bind_rows(data_sra, data_sra_missing)
tmp <- data_sra$n
tmp <- replace_na(tmp, 0)
data_sra$nn <- tmp

sra_num_samples_fig <- data_sra %>% 
  ggplot(aes(x = bin, y = factor(tissue, levels = sra_data_tissue_order), fill = factor(n))) +
  geom_tile() +
  geom_text(aes(label = nn), color = "white") +
  xlab("Experiments per sample size range") +
  #scale_fill_viridis_c() +
  scale_fill_brewer(palette = "Blues", na.value = "light gray") +
  theme(panel.background = element_blank(),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_blank(),
        text = element_text(size = 14),
        axis.title.y = element_blank(),
        axis.title.x = element_text(size = 10),
        legend.position = "none")
```

```{r echo=FALSE, fig.cap="**Data.** The barplot shows the number of experiments from each tissue in the SRA data. Blue bars indicate tissues for which we were able to create a tissue-aware gold standard. Tissues with gray bars were evaluated on the tissue-naive standard only. The heatmap on the right shows the number of projects/experiments that have a particular sample size for each tissue.", out.width="100%"}
sra_projects_per_tissue_fig + sra_num_samples_fig
```

# Overall Method Performance  {.tabset .tabset-fade .tabset-pills}

## log<sub>2</sub>(auPRC/prior)
```{r echo=FALSE, fig.cap="**Overall performance of workflows.** The plots show the aggregate accuracy of all SRA coexpression networks resulting from each individual workflow evaluated using the (**left**) tissue-naive and (**right**) tissue-aware gold standards. The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. The performance of each workflow is presented as boxplots that summarizes the log2(auPRC/prior) of each workflow where auPRC is the area under the precision recall curve. The workflows are ordered by their median log2(auPRC/prior) for the tissue-naive data.", out.width="100%"}
knitr::include_graphics("plots/sra_auprc_box.png")
```

## auROC
```{r echo=FALSE, fig.cap="**Overall performance of workflows.** The plots show the aggregate accuracy of all SRA coexpression networks resulting from each individual workflow evaluated using the (**left**) tissue-naive and (**right**) tissue-aware gold standards. The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. The performance of each workflow is presented as boxplots that summarizes the auROC of each workflow where auPRC is the area under the precision recall curve. The workflows are ordered by their median auROC for the tissue-naive data.", out.width="100%"}
knitr::include_graphics("plots/sra_auroc_box.png")
```

## log<sub>2</sub>(p20r/prior)
```{r echo=FALSE, fig.cap="**Overall performance of workflows.** The plots show the aggregate accuracy of all SRA coexpression networks resulting from each individual workflow evaluated using the (**left**) tissue-naive and (**right**) tissue-aware gold standards. The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. The performance of each workflow is presented as boxplots that summarizes the log2(p20r/prior) of each workflow where auPRC is the area under the precision recall curve. The workflows are ordered by their median log2(p20r/prior) for the tissue-naive data.", out.width="100%"}
knitr::include_graphics("plots/sra_p20r_box.png")
```

# Proportion of Times Each Method Outperforms Another {.tabset .tabset-fade .tabset-pills}

## log<sub>2</sub>(auPRC/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, out.width = '49%'}
knitr::include_graphics("plots/heatmap_legend.png")
```

```{r echo=FALSE, fig.cap="**Dataset-level pairwise comparison of workflow performance.** (**left**) The heatmap shows the relative performance of a pair of workflows, corresponding to a row and a column, directly compared to each other for the SRA datasets based on the tissue-naive gold standard. The color in each cell (row, column) represents the proportion of datasets for which the workflow along the row has a higher log2(auPRC/prior) than the workflow along the column. Comparisons that are statistically significant (corrected p < 0.01) based on a paired Wilcoxon test are marked with an asterisk. (**middle**) The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. (**right**) The barplot shows the number of times each workflow was significantly greater than another workflow.", out.width = '100%'}
knitr::include_graphics("plots/sra_auprc_naive_heatmap.png")
```
### Tissue-aware Standard
```{r echo=FALSE, out.width = '49%'}
knitr::include_graphics("plots/heatmap_legend.png")
```

```{r echo=FALSE, fig.cap="**Dataset-level pairwise comparison of workflow performance.** (**left**) The heatmap shows the relative performance of a pair of workflows, corresponding to a row and a column, directly compared to each other for the SRA datasets based on the tissue-aware gold standard. The color in each cell (row, column) represents the proportion of datasets for which the workflow along the row has a higher log2(auPRC/prior) than the workflow along the column. Comparisons that are statistically significant (corrected p < 0.01) based on a paired Wilcoxon test are marked with an asterisk. (**middle**) The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. (**right**) The barplot shows the number of times each workflow was significantly greater than another workflow.", out.width = '100%'}
knitr::include_graphics("plots/sra_auprc_knowledge_heatmap.png")
```

## auROC {.tabset .tabset-fade .tabset-pills}
### Tissue-naive Standard
```{r echo=FALSE, out.width = '49%'}
knitr::include_graphics("plots/heatmap_legend.png")
```

```{r echo=FALSE, fig.cap="**Dataset-level pairwise comparison of workflow performance.** (**left**) The heatmap shows the relative performance of a pair of workflows, corresponding to a row and a column, directly compared to each other for the SRA datasets based on the tissue-naive gold standard. The color in each cell (row, column) represents the proportion of datasets for which the workflow along the row has a higher auROC than the workflow along the column. Comparisons that are statistically significant (corrected p < 0.01) based on a paired Wilcoxon test are marked with an asterisk. (**middle**) The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. (**right**) The barplot shows the number of times each workflow was significantly greater than another workflow.", out.width = '100%'}
knitr::include_graphics("plots/sra_auroc_naive_heatmap.png")
```
### Tissue-aware Standard
```{r echo=FALSE, out.width = '49%'}
knitr::include_graphics("plots/heatmap_legend.png")
```

```{r echo=FALSE, fig.cap="**Dataset-level pairwise comparison of workflow performance.** (**left**) The heatmap shows the relative performance of a pair of workflows, corresponding to a row and a column, directly compared to each other for the SRA datasets based on the tissue-aware gold standard. The color in each cell (row, column) represents the proportion of datasets for which the workflow along the row has a higher auROC than the workflow along the column. Comparisons that are statistically significant (corrected p < 0.01) based on a paired Wilcoxon test are marked with an asterisk. (**middle**) The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. (**right**) The barplot shows the number of times each workflow was significantly greater than another workflow.", out.width = '100%'}
knitr::include_graphics("plots/sra_auroc_knowledge_heatmap.png")
```

## log<sub>2</sub>(p20r/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, out.width = '49%'}
knitr::include_graphics("plots/heatmap_legend.png")
```

```{r echo=FALSE, fig.cap="**Dataset-level pairwise comparison of workflow performance.** (**left**) The heatmap shows the relative performance of a pair of workflows, corresponding to a row and a column, directly compared to each other for the SRA datasets based on the tissue-naive gold standard. The color in each cell (row, column) represents the proportion of datasets for which the workflow along the row has a higher log2(p20r/prior) than the workflow along the column. Comparisons that are statistically significant (corrected p < 0.01) based on a paired Wilcoxon test are marked with an asterisk. (**middle**) The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. (**right**) The barplot shows the number of times each workflow was significantly greater than another workflow.", out.width = '100%'}
knitr::include_graphics("plots/sra_p20r_naive_heatmap.png")
```
### Tissue-aware Standard
```{r echo=FALSE, out.width = '49%'}
knitr::include_graphics("plots/heatmap_legend.png")
```

```{r echo=FALSE, fig.cap="**Dataset-level pairwise comparison of workflow performance.** (**left**) The heatmap shows the relative performance of a pair of workflows, corresponding to a row and a column, directly compared to each other for the SRA datasets based on the tissue-aware gold standard. The color in each cell (row, column) represents the proportion of datasets for which the workflow along the row has a higher log2(p20r/prior) than the workflow along the column. Comparisons that are statistically significant (corrected p < 0.01) based on a paired Wilcoxon test are marked with an asterisk. (**middle**) The workflows (rows) are described in terms of the specific method used in the within-sample normalization (blues), between-sample normalization (greens), and network transformation (oranges) stages. (**right**) The barplot shows the number of times each workflow was significantly greater than another workflow.", out.width = '100%'}
knitr::include_graphics("plots/sra_p20r_knowledge_heatmap.png")
```

# Method Part Comparison {.tabset .tabset-fade .tabset-pills}

## log<sub>2</sub>(auPRC/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, fig.cap="**Impact of individual methods on performance of workflows.** Each bar in the barplot corresponds to a specific method, showing the proportion of times (x-axis) that workflows including that particular method (y-axis) were significantly better than other workflows. In order to make the comparison of between-sample normalization methods fair, workflows also including CPM, RPKM, or TPM were left out because it is not possible to pair them with TMM or UQ normalization. Similarly, TMM and UQ methods are not included for “no within-sample normalization” (NO–WI).", out.width="100%"}
knitr::include_graphics("plots/sra_naive_auprc_bar.png")
```

### Tissue-aware Standard
```{r echo=FALSE, fig.cap="**Impact of individual methods on performance of workflows.** Each bar in the barplot corresponds to a specific method, showing the proportion of times (x-axis) that workflows including that particular method (y-axis) were significantly better than other workflows. In order to make the comparison of between-sample normalization methods fair, workflows also including CPM, RPKM, or TPM were left out because it is not possible to pair them with TMM or UQ normalization. Similarly, TMM and UQ methods are not included for “no within-sample normalization” (NO–WI).", out.width="100%"}
knitr::include_graphics("plots/sra_knowledge_auprc_bar.png")
```

## auROC {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, fig.cap="**Impact of individual methods on performance of workflows.** Each bar in the barplot corresponds to a specific method, showing the proportion of times (x-axis) that workflows including that particular method (y-axis) were significantly better than other workflows. In order to make the comparison of between-sample normalization methods fair, workflows also including CPM, RPKM, or TPM were left out because it is not possible to pair them with TMM or UQ normalization. Similarly, TMM and UQ methods are not included for “no within-sample normalization” (NO–WI).", out.width="100%"}
knitr::include_graphics("plots/sra_naive_auroc_bar.png")
```
### Tissue-aware Standard
```{r echo=FALSE, fig.cap="**Impact of individual methods on performance of workflows.** Each bar in the barplot corresponds to a specific method, showing the proportion of times (x-axis) that workflows including that particular method (y-axis) were significantly better than other workflows. In order to make the comparison of between-sample normalization methods fair, workflows also including CPM, RPKM, or TPM were left out because it is not possible to pair them with TMM or UQ normalization. Similarly, TMM and UQ methods are not included for “no within-sample normalization” (NO–WI).", out.width="100%"}
knitr::include_graphics("plots/sra_knowledge_auroc_bar.png")
```

## log<sub>2</sub>(p20r/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, fig.cap="**Impact of individual methods on performance of workflows.** Each bar in the barplot corresponds to a specific method, showing the proportion of times (x-axis) that workflows including that particular method (y-axis) were significantly better than other workflows. In order to make the comparison of between-sample normalization methods fair, workflows also including CPM, RPKM, or TPM were left out because it is not possible to pair them with TMM or UQ normalization. Similarly, TMM and UQ methods are not included for “no within-sample normalization” (NO–WI).", out.width="100%"}
knitr::include_graphics("plots/sra_naive_p20r_bar.png")
```
### Tissue-aware Standard
```{r echo=FALSE, fig.cap="**Impact of individual methods on performance of workflows.** Each bar in the barplot corresponds to a specific method, showing the proportion of times (x-axis) that workflows including that particular method (y-axis) were significantly better than other workflows. In order to make the comparison of between-sample normalization methods fair, workflows also including CPM, RPKM, or TPM were left out because it is not possible to pair them with TMM or UQ normalization. Similarly, TMM and UQ methods are not included for “no within-sample normalization” (NO–WI).", out.width="100%"}
knitr::include_graphics("plots/sra_knowledge_p20r_bar.png")
```

# Impact of Sample Size {.tabset .tabset-fade .tabset-pills}
Sample size refers to the number of samples used to create a network. Each barplot shows the Spearman correlation of sample size and performance for each workflow.

## log<sub>2</sub>(auPRC/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_auprc_sample_size_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_auprc_sample_size_plot.png")
```

## auROC {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_auroc_sample_size_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_auroc_sample_size_plot.png")
```

## log<sub>2</sub>(p20r/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_p20r_sample_size_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_p20r_sample_size_plot.png")
```

# Impact of Sample Similarity {.tabset .tabset-fade .tabset-pills}
Sample similarity is determined by subsetting all samples to the 50% most variable genes in the experiment, then calculating the spearman correlation between all sample pairs in the experiment and taking the median value. Each barplot shows the Spearman correlation of sample similarity and performance for each workflow. 

## log<sub>2</sub>(auPRC/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_auprc_sample_sim_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_auprc_sample_sim_plot.png")
```

## auROC {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_auroc_sample_sim_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_auroc_sample_sim_plot.png")
```

## log<sub>2</sub>(p20r/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_p20r_sample_sim_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_p20r_sample_sim_plot.png")
```

# Impact of Read Count Diversity {.tabset .tabset-fade .tabset-pills}
Read count diversity is calulated for a given experiment by taking the standard deviation of the sum of the counts in each sample. Each barplot shows the Spearman correlation of read count diversity and performance for each workflow.

## log<sub>2</sub>(auPRC/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_auprc_count_div_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_auprc_count_div_plot.png")
```

## auROC {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_auroc_count_div_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_auroc_count_div_plot.png") 
```

## log<sub>2</sub>(p20r/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_naive_p20r_count_div_plot.png")
```

### Tissue-aware Standard

```{r echo=FALSE, message=FALSE}
knitr::include_graphics("plots/sra_knowledge_p20r_count_div_plot.png")
```

# Impact of Tissue {.tabset .tabset-fade .tabset-pills}
Each point in the interactive scatterplots represents the median performance (y-axis) of each workflow (color of the point) in each tissue (x-axis).

## log<sub>2</sub>(auPRC/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, message=FALSE}
l2_auprc_prior_tis_plot <- snr %>% 
  select(method, tissue, log2_auprc_over_prior) %>% 
  group_by(method, tissue) %>% 
  summarise(median_perf = median(log2_auprc_over_prior)) %>% 
  ggplot(aes(x = tissue, y = median_perf, color = method)) +
  geom_point() +
  ylab("median log2(auPRC/prior)") +
  theme(panel.background = element_rect(fill = "white"), 
        panel.grid.major = element_line(colour = "#999999", size = 0.1),
        panel.grid.minor = element_line(colour = "#999999", size = 0.1),
        axis.text.x = element_text(angle = 90),
        axis.title.x = element_blank(),
        text = element_text(size = 14),
        legend.key = element_blank())

ggplotly(l2_auprc_prior_tis_plot)
```

### Tissue-aware Standard
```{r echo=FALSE, message=FALSE}
l2_auprc_prior_tis_plot_ts <- skr %>% 
  select(method, tissue, log2_auprc_over_prior) %>% 
  group_by(method, tissue) %>% 
  summarise(median_perf = median(log2_auprc_over_prior)) %>% 
  ggplot(aes(x = tissue, y = median_perf, color = method)) +
  geom_point() +
  ylab("median log2(auPRC/prior)") +
  theme(panel.background = element_rect(fill = "white"), 
        panel.grid.major = element_line(colour = "#999999", size = 0.1),
        panel.grid.minor = element_line(colour = "#999999", size = 0.1),
        axis.text.x = element_text(angle = 90),
        axis.title.x = element_blank(),
        text = element_text(size = 14),
        legend.key = element_blank())

ggplotly(l2_auprc_prior_tis_plot_ts)
```

## auROC {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, message=FALSE}
l2_auroc_tis_plot <- snr %>% 
  select(method, tissue, auROC) %>% 
  group_by(method, tissue) %>% 
  summarise(median_perf = median(auROC)) %>% 
  ggplot(aes(x = tissue, y = median_perf, color = method)) +
  geom_point() +
  ylab("median auROC") +
  theme(panel.background = element_rect(fill = "white"), 
        panel.grid.major = element_line(colour = "#999999", size = 0.1),
        panel.grid.minor = element_line(colour = "#999999", size = 0.1),
        axis.text.x = element_text(angle = 90),
        axis.title.x = element_blank(),
        text = element_text(size = 14),
        legend.key = element_blank())

ggplotly(l2_auroc_tis_plot)
```

### Tissue-aware Standard
```{r echo=FALSE, message=FALSE}
l2_auroc_tis_plot_ts <- skr %>% 
  select(method, tissue, auROC) %>% 
  group_by(method, tissue) %>% 
  summarise(median_perf = median(auROC)) %>% 
  ggplot(aes(x = tissue, y = median_perf, color = method)) +
  geom_point() +
  ylab("median auROC") +
  theme(panel.background = element_rect(fill = "white"), 
        panel.grid.major = element_line(colour = "#999999", size = 0.1),
        panel.grid.minor = element_line(colour = "#999999", size = 0.1),
        axis.text.x = element_text(angle = 90),
        axis.title.x = element_blank(),
        text = element_text(size = 14),
        legend.key = element_blank())

ggplotly(l2_auroc_tis_plot_ts)
```

## log<sub>2</sub>(p20r/prior) {.tabset .tabset-fade .tabset-pills}

### Tissue-naive Standard
```{r echo=FALSE, message=FALSE}
l2_prec_at_recall20_tis_plot <- snr %>% 
  select(method, tissue, log2_p20r) %>% 
  group_by(method, tissue) %>% 
  summarise(median_perf = median(log2_p20r)) %>% 
  ggplot(aes(x = tissue, y = median_perf, color = method)) +
  geom_point() +
  ylab("median log2(p20r/prior)") +
  theme(panel.background = element_rect(fill = "white"), 
        panel.grid.major = element_line(colour = "#999999", size = 0.1),
        panel.grid.minor = element_line(colour = "#999999", size = 0.1),
        axis.text.x = element_text(angle = 90),
        axis.title.x = element_blank(),
        text = element_text(size = 14),
        legend.key = element_blank())

ggplotly(l2_prec_at_recall20_tis_plot)
```

### Tissue-aware Standard
```{r echo=FALSE, message=FALSE}
l2_prec_at_recall20_tis_plot_ts <- skr %>% 
  select(method, tissue, log2_p20r) %>% 
  group_by(method, tissue) %>% 
  summarise(median_perf = median(log2_p20r)) %>% 
  ggplot(aes(x = tissue, y = median_perf, color = method)) +
  geom_point() +
  ylab("median log2(p20r/prior)") +
  theme(panel.background = element_rect(fill = "white"), 
        panel.grid.major = element_line(colour = "#999999", size = 0.1),
        panel.grid.minor = element_line(colour = "#999999", size = 0.1),
        axis.text.x = element_text(angle = 90),
        axis.title.x = element_blank(),
        text = element_text(size = 14),
        legend.key = element_blank())

ggplotly(l2_prec_at_recall20_tis_plot_ts)
```